Dockerfile

```diff
@@ -2,26 +2,20 @@ FROM node:18.14.0-alpine3.16 as asset-env
 
 WORKDIR /app
 
-RUN mkdir -p web/static
+COPY package.json yarn.lock ./
+RUN yarn --prod
 
-COPY package.json .
-COPY yarn.lock .
-RUN yarn
-
-COPY app/web/assets web/assets
-RUN yarn build
+COPY web/assets web/assets
+RUN mkdir -p web/static && yarn build
 
 FROM golang:1.20 as build-env
 
 WORKDIR /app
 
-COPY app/go.mod .
-COPY app/go.sum .
-
+COPY app/go.mod app/go.sum ./
 RUN go mod download
 
 COPY /app .
-
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -installsuffix cgo -o /go/bin/mlpab
 
 FROM build-env as development
```

Comment:
I don't think anyone is actually using air now so we can drop the whole `development` build env

Reply:
:+1: cool